apps/src/applab/commands.js

```diff
@@ -554,6 +554,19 @@ applabCommands.setStrokeWidth = function (opts) {
   return false;
 };
 
+// Returns an rbg or rgba color string that can be used as a parameter to other functions.
+applabCommands.rgb = function (opts) {
+  apiValidateTypeAndRange(opts, 'color', 'number', opts.r, 'number', 0, 255);
+  apiValidateTypeAndRange(opts, 'color', 'number', opts.g, 'number', 0, 255);
+  apiValidateTypeAndRange(opts, 'color', 'number', opts.b, 'number', 0, 255);
+  var colorString = opts.r + "," + opts.g + "," + opts.b;
```

Comment:
I know we dont really use much ES6 in this file, but we could choose to write this as:
```
const colorString = `${opts.r}, ${opts.g}, ${opts.b}`;
```
Which reads slightly better imo. Same is true of lines 565, 567. Up to you if you want to bother changing.